www/davj05/cv04/admin/users.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require __DIR__ . "../functions.php";
```

Comment:
![image](https://github.com/nvbach91/4IZ278-2023-2024-LS/assets/20724910/3dbbba8e-6f1a-4e92-a2c0-ad7132c2898b)

Reply:
Díky, kouknu na to.